app/serializers/topic_list_item_serializer.rb

```diff
@@ -142,6 +143,10 @@ def include_allowed_user_count?
     object.private_message?
   end
 
+  def is_hot
+    TopicHotScore.hottest_topic_ids.to_a.include?(object.id)
```

Comment:
```suggestion
    TopicHotScore.hottest_topic_ids.include?(object.id)
```

`to_a` is not necessary since `TopicHotScore.hottest_topic_ids` already returns an array. `include?` will still work with a `Set` as well.